frontend/src/components/homepage/stats.js

```diff
@@ -41,7 +41,7 @@ export const StatsSection = () => {
     const abortController = new AbortController();
     // Using axios over the useFetch hook for external API endpoint
     const fetchOsmStats = () =>
-      axios.get(HOMEPAGE_STATS_API_URL, {
+      axios.get(`${OHSOME_STATS_BASE_URL}/stats/hotosm-project-%2A?ohsomeFormat=false`, {
```

Comment:
here you can remove the `ohsomeFormat=false` parameter. It is not in use by the API anymore. Removing it will not change the response format.

Reply:
Thank you for pointing out the unnecessary parameter!